src/api.js

```diff
@@ -0,0 +1,49 @@
+const UserInputUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/z7JM1rmeo6NjLkeyvLYM/scores/';
+const gameNameUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+const createGameName = async () => {
+  const gameName = 'Superheroes Rescue';
+  const response = await fetch(gameNameUrl, {
+    method: 'POST',
+    body: JSON.stringify({ name: gameName }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  const gameData = await response.json();
+  return gameData;
+};
```

Comment:
Great job implementing  `async / await` function to create a game object.. with predefined name of your choice 👍🏽